app/models/dialog_field_serializer.rb

```diff
@@ -25,6 +25,13 @@ def serialize(dialog_field, all_attributes = false)
       category = Category.find_by(:id => dialog_field.category)
       dialog_field.options.merge!(:category_name => category.name, :category_description => category.description)
     end
-    included_attributes(dialog_field.as_json(:methods => [:type, :values]), all_attributes).merge(extra_attributes)
+
+    field_as_json = dialog_field.as_json(:methods => [:type, :values])
+    field_as_json["default_value"] = begin
+      JSON.parse(field_as_json["default_value"])
+    rescue
+      field_as_json["default_value"]
+    end
```

Comment:
I don't know if we need a begin/rescue chain, the only way that the default value would be a list of items would be if `dialog_field.force_multi_value` returns true.

Also, if the `default_value` is just a string with comma separated values, won't `JSON.parse` just fail? If you want to serialize this as an array you'll have to manually split it up, add it to a temporary array and then `.to_json` the array.